modules/planning/tasks/deciders/path_bounds_decider.h

```diff
@@ -114,7 +114,8 @@ class PathBoundsDecider : public Decider {
   bool SearchPullOverPosition(
       const ReferenceLineInfo& reference_line_info,
       const std::vector<std::tuple<double, double, double>>& path_bound,
-      std::tuple<double, double, double>* const pull_over_configuration);
+      std::tuple<double, double, double, double, double>* const
```

Comment:
Long tuples are hard to read and maintain. Use struct or proto instead.
But it's OK leave as it is for this PR.

Reply:
Yes, that is why I left a TODO for me and jiacheng for a re-factor